report/renderer/performance-category-renderer.js

```diff
@@ -200,7 +220,6 @@ export class PerformanceCategoryRenderer extends CategoryRenderer {
       showEl.textContent = Util.i18n.strings.expandView;
       hideEl.textContent = Util.i18n.strings.collapseView;
 
-      const metricAudits = category.auditRefs.filter(audit => audit.group === 'metrics');
```

Comment:
Took me a half an hour to figure out why it wasn't working. Turns out we set `metricsAudits` twice.